doc/1_applications.dox

```diff
@@ -0,0 +1,163 @@
+/**
+
+@page applications Sample Applications
+
+@tableofcontents
+
+@section application_introduction Introduction
+
+The easiest way to get started for many people is with samples. We've provided
+the source for some applications. Some of these applications you can run
+directly from the command line. While others will require a device.
+
+@section application_folders Directory Organization
+
+The sample applications typically have the following folder structure:
+
+    targets/
+    config.mk
+    Makefile
+    subdirs
+
+The code for an application typcially consists of two parts: shared and
+target-specific. The shared code will be at the top folder for the application.
+While the target-specific information will be under a target folder. As an
+example, the blink_raw application has shared code here:
```

Comment:
the blink_raw application has these files among the shared and target-specific code, respectively: